src/config/schema.types.d.ts

```diff
@@ -53,6 +53,14 @@ type YamlSchema = {
     onChainWallet: string
     targetConfirmations: number
   }
+  bria: {
+    hotWalletName: string
+    queues: {
+      fast: string
+      medium: string
+      slow: string
+    }
```

Comment:
@bodymindarts this is what you mean with inject queues? I thought was inject a list of queues